requirements.txt

```diff
@@ -5,7 +5,7 @@ freezegun~=0.3.15
 google-api-core[grpc] >= 1.14.0, < 2.0.0dev
 matplotlib~=3.0
 networkx~=2.4
-numpy~=1.16, < 1.19
+numpy>=1.19.1
```

Comment:
Thanks for opening this! 
I would prefer keeping it `~=1.19.1` so that we don't get auto-upgraded suddenly to the next minor version.

Reply:
My bad, modified! A side note though, the previous `numpy>=1.19.1` would somehow break the pytest ubuntu and the coverage test too. I have tested on my machine which runs manjaro with python 3.7 and 3.6 with numpy of 1.19.1. No error shows up when I run check/all command.

Reply:
Interesting! We should get to the bottom of this - do you mind testing a couple of times with `>=1.19.1` and `~=` by pushing different versions? It shouldn't matter at this moment!!! 

It seems that this is a flaky situation but the [failing build on your previous commit](https://github.com/quantumlib/Cirq/runs/982916285) did reproduce https://github.com/quantumlib/Cirq/issues/3103. We shouldn't merge until we confirm that there is no flake - i.e. if you can run this test on your ubuntu machine a 100 times and it won't fail, then we are confident that it works.

Reply:
Oh one more thing - the best course of action in case of flakes is to understand the root cause of why these flakes are happening and then resolve that. If you can pinpoint the reason why that test failed in the first place - that would give us enough confidence to upgrade to numpy 1.19+ without introducing flakes. Thank you for your investigations on this!

Reply:
> Oh one more thing - the best course of action in case of flakes is to understand the root cause of why these flakes are happening and then resolve that. If you can pinpoint the reason why that test failed in the first place - that would give us enough confidence to upgrade to numpy 1.19+ without introducing flakes. Thank you for your investigations on this!

Got it, will investigate later!